data/src/main/java/me/androidbox/data/remote/authentication/AuthenticationImp.kt

```diff
@@ -0,0 +1,37 @@
+package me.androidbox.data.remote.authentication
+
+import me.androidbox.data.remote.model.request.LoginRequestDto
+import me.androidbox.data.remote.model.request.RegistrationRequestDto
+import me.androidbox.data.remote.network.authentication.AuthenticationService
+import me.androidbox.domain.authentication.model.LoginModel
+import me.androidbox.domain.authentication.remote.Authentication
+import javax.inject.Inject
+
+class AuthenticationImp @Inject constructor(
```

Comment:
Is this the repository?

If so, it's common practice to have the word Repository in it
Ex: AuthenticationRepositoryImpl

Reply:
@MKiperszmid @philipplackner 
This is requesting and fetching from the API endpoint. What would the best naming be for this?